companyweb_base/models/res_partner.py

```diff
@@ -518,7 +518,7 @@ def cweb_button_enhance(self):
         )
         if r["StatusCode"] in [101, 302]:
             return self._cweb_call_wizard_credentials("Enter Companyweb credentials")
-        elif r["StatusCode"] != 0:
+        if r["StatusCode"] != 0:
```

Comment:
@xavier-bouquiaux Why changing elif here in the migration commit ?

Reply:
I changed this to keep the code aligned across versions